config/development.json

```diff
@@ -70,6 +70,7 @@
 		"i18n/community-translator": false,
 		"i18n/empathy-mode": true,
 		"i18n/translation-scanner": true,
+		"importer/unified": true,
```

Comment:
My opinion is that let's default this to `false` as otherwise testing imports outside our team will require a negative flag, and that needs communication. Let's make this true when we have it working.

Reply:
I agree, changed.